pom.xml

```diff
@@ -154,14 +154,14 @@
 					<target>1.7</target>
 				</configuration>
 			</plugin>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-jar-plugin</artifactId>
-				<version>2.3.1</version>
-				<configuration>
-					<outputDirectory>C:\Users\Arte\Desktop\Server 1.13\plugins</outputDirectory>
-				</configuration>
-			</plugin>
+<!-- 			<plugin> -->
+<!-- 				<groupId>org.apache.maven.plugins</groupId> -->
+<!-- 				<artifactId>maven-jar-plugin</artifactId> -->
+<!-- 				<version>2.3.1</version> -->
+<!-- 				<configuration> -->
+<!-- 					<outputDirectory>C:\Users\Arte\Desktop\Server\plugins</outputDirectory> -->

```

Comment:
_C:\Users\Arte\Desktop\Server\plugins_ **Change to** _C:\Users\Arte\Desktop\Server 1.13\plugins_